runtime/src/common/sgx/pcs.rs

```diff
@@ -244,13 +244,14 @@ impl QuoteBundle {
     }
 }
 
-struct Verifier {
+/// Quoting Enclave ECDSA P-256 verifier.
+pub struct QeEcdsaP256Verifier {
```

Comment:
You probably also want to make the fields public as otherwise you won't be able to construct it? Or make a constructor (e.g. `new(tcb_info, qe_identity)`) and an accessor for retrieving the `tcb_level`.

Reply:
Thats right, updated!